tasks/generate_interfaces.yml

```diff
@@ -71,6 +71,7 @@
 - name: Create random temporary directory for ifupdown data
   command: mktemp -d
   register: ifupdown_register_tempdir
+  always_run: True
```

Comment:
Does this mean that the temporary directory is created every time, even when there are no changes expected?